pom.xml

```diff
@@ -4,61 +4,74 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>3.57</version>
+		<version>4.75</version>
 		<relativePath />
 	</parent>
+
 	<artifactId>matlab</artifactId>
 	<version>2.15.1-SNAPSHOT</version>
 	<packaging>hpi</packaging>
+
+	<name>MATLAB Plugin</name>
+	<url>https://github.com/jenkinsci/matlab-plugin</url>
+	<description>Jenkins plugin for MATLAB</description>
+
 	<developers>
 		<developer>
 			<id>mathworks_ci_team</id>
 			<name>MathWorks</name>
 			<email>nbhoski@mathworks.com</email>
```

Comment:
We might consider updating this to the ci email

Reply:
That's probably smart, that will definitely reach us faster.

Reply:
Yeah I updated it to continuous-integration@mathworks.com. I think I remember talking to Nikhil about it and he said the Jenkins team preferred to have an individuals email over a mailing list, but we'll update it and see how it goes.